aries-site/src/examples/components/layouts/FooterExample.js

```diff
@@ -14,21 +13,19 @@ export const FooterExample = () => {
   return (
     <Footer
       background="background-front"
-      direction={size !== 'small' ? 'row' : 'column-reverse'}
+      direction="row-responsive"
```

Comment:
In mobile, this now presents links below the copyright. Whereas previously, the designs called for links above the copyright. Is this change what we want?

Reply:
Yes, we do want links below. However, I'm seeing differences now between Chris's sticker sheet footer and Greg's library footer. From my discussion last week with Chris, we should use his sticker sheet footer as reference.
https://www.figma.com/file/K0PppsSh6aQSszqlwKAekg/hpe-design-system-stickers?node-id=382%3A259

Reply:
We should make sure when I asked Greg said to use his Figma file and not Chris sticker sheet

Reply:
I'll send them a message

Reply:
Update: We should use the library file (not the sticker sheet for design guidance). I'll be updating this PR.